.github/ISSUE_TEMPLATE/design_proposal.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 🤔 Design proposal
+about: Collaborate on a design for a feature/fix or other change
+labels: design-proposal
+---
+
+<!--
+This template is useful to build consensus about whether work should be done, and if so, the high-level shape of how it should be approached. Use this before fixating on a particular implementation.
+-->
+
+## Summary
+
+1-2 sentences. Say what this is about.
+
+## Motivation and goals
```

Comment:
nit: can we change goals to "success criteria" I've found this phrasing generally results in more specific and measurable goals/criteria